yargs.js

```diff
@@ -33,6 +33,8 @@ function Yargs (processArgs, cwd, parentRequire) {
     updateFiles: false
   })
 
+  self.processArgs = processArgs.slice()
```

Comment:
What problem does this solve? You don't have access to `process.argv`?

Reply:
processArgs is from args pass by Yargs (processArgs, cwd, parentRequire)

this is not process.argv